rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java

```diff
@@ -65,6 +70,35 @@ public static TableMetaCache getInstance() {
         return tableMetaCache;
     }
 
+    @Override
+    protected String getCacheKey(DataSourceProxy dataSourceProxy, String tableName) {
+        StringBuilder cacheKey = new StringBuilder(dataSourceProxy.getResourceId());
+        cacheKey.append(".");
+        String defaultTableName = tableName.replace("`", "");
+
+        DatabaseMetaData databaseMetaData = null;
+        try {
+            databaseMetaData = dataSourceProxy.getPlainConnection().getMetaData();
+        } catch (SQLException e) {
+            LOGGER.error("Could not get connection, use default cache key", e.getMessage(), e);
+            return cacheKey.append(cacheKey.append(defaultTableName)).toString();
```

Comment:
Why append itself?Is it cacheKey.append(defaultTableName).toString();

Reply:
Thanks. Fixed.